README.md

```diff
@@ -1,13 +1,16 @@
 # Project Todos
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This is a to do application that uses local storage to save your very own to do list. There are two tasks already created at the start to show the design of the posts, but thanks to local storage, once you delete them they won't come back. It's also possible to add new to dos as well as delete created ones. You also get a date stamp when the to do was created. When you check a to do as complete it will be sorted to the end of the list of all tasks.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+The tools used for this project was redux and it's my first project writing reducers. What I like most about the project is the use of redux localstorage which had a very quick implementation.
+I'm using a combined css and styled components, the styling for the checkboxes I would like to adapt from css to a styled component to have all the styling in one place.
```

Comment:
You are not alone on this one -  did that too - couldn't figure out how to use styled components for the checkbox.